thunder/tests/distributed/test_dtensor.py

```diff
@@ -136,6 +136,54 @@ def fn(x, w):
         with pytest.raises(RuntimeError, match="has changed for cotangent between tracing and runtime"):
             torch.autograd.grad(actual, (in_dtensor, w_dtensor), g_o)
 
+    @common_utils.parametrize("executor", tuple(executors_map.keys()))
+    def test_dtensor_convert_element_type(self, executor):
```

Comment:
Just curious, can we use executors defined in `thunder.tests.framework` here?

Reply:
`executors_map` also uses the same executors defined in `thunder.tests.framework`.

https://github.com/Lightning-AI/lightning-thunder/blob/cbfaf03716f6a11f1060b017ee75e3420a511206/thunder/tests/distributed/helper.py#L43-L47

At the moment, DTensor doesn't with DynamoExecutor when backward is also required (see https://github.com/Lightning-AI/lightning-thunder/issues/2234).

Reply:
I think I was curious if `@common_utils.parametrize("executor", tuple(executors_map.values()))`

Reply:
That works, but then the test name would be `test_dtensor_convert_element_type_executor0`. So, we can either add `ids` to `parametrize` or pass the executor names like we have now.